django/db/backends/base/references.py

```diff
@@ -0,0 +1,129 @@
+"""
+Helpers to manipulate DDL statements that might need to be adjusted or
+discarded within a schema edition lifecycle.
```

Comment:
I'm not sure what "schema edition lifecycle" means. Is "edition" the correct word choice?